_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -21,39 +21,33 @@ acknowledgments:
+- **hidden**: the element is not [visible][], and not [included in the accessibility tree][]; or
+- **disabled**: the element has an `aria-disabled` [attribute value][] of `true`; or
+- **fixed value**: the element is an `input` element with a `type` [attribute value][] of either `hidden`, `button`, `submit` or `reset`; or
+- **static**: the element is not part of [sequential focus navigation][] and has a [semantic role][] that is not a [widget role][].
 
 ## Expectation 1
 
-The `autocomplete` attribute is a single term, or a space separated list of terms.
+Each test target's `autocomplete` [attribute value][] is a [space separated][] list of one or more tokens that follow the [HTML specification for Autofill detail tokens][], which requires that the token list match the following in the correct order:
 
-## Expectation 2
-
-The autocomplete term(s) follow the [HTML specification - Autofill detail tokens](https://html.spec.whatwg.org/#autofill-detail-tokens), which requires that it/they match the following in the correct order:
-
-1. Has a value that starts with "section-" _(optional)_
-2. Has either "shipping" or "billing" _(optional)_
-3. Has either "home", "work", "mobile", "fax" or "pager" _(optional, only for "email", "impp", "tel" or "tel-\*")_
-4. Has a [correct autocomplete field](#correct-autocomplete-field) _(required)_
-
-**Note:** Autocomplete terms are case insensitive. When multiple terms are used, they must be used in the correct order.
+1. An optional token that starts with "section-"; then
+2. An optional token of either "shipping" or "billing"; then
+3. An optional token of either "home", "work", "mobile", "fax" or "pager", only if the last token is "email", "impp", "tel" or "tel-*"; then
+4. A required token from the [correct autocomplete field][].
 
-## Expectation 3
+## Expectation 2
 
-The [correct autocomplete field](#correct-autocomplete-field) is an [appropriate field for the form control](#appropriate-field-for-the-form-control).
+Each test target's `autocomplete` [attribute value][] has a [correct autocomplete field][] that is an [appropriate][appropriate field for the form control] for that test target.
 
 ## Assumptions
 
-For this rule, it is assumed that the `autocomplete` attribute is not used on form fields that do not correspond to an autocomplete field described in the HTML 5.2 specification. If the `autocomplete` field is used to describe "custom" taxonomy, rather than that described in the specification, this rule may produce incorrect results.
+The `autocomplete` attribute is not used on form fields that do not correspond to an autocomplete field described in the HTML 5.2 specification. If the `autocomplete` field is used to describe "custom" taxonomy, rather than that described in the specification, Success Criterion [1.3.5 Identify Input Purpose][sc135] may be satisfied even if this rule failed.
```

Comment:
```suggestion
The `autocomplete` attribute is not used on form fields that do not correspond with an autocomplete field described in the HTML 5.2 specification. If the `autocomplete` field is used to describe "custom" taxonomy, rather than that described in the specification, Success Criterion [1.3.5 Identify Input Purpose][sc135] may be satisfied even if this rule failed.
```

Reply:
Feels more accurate to me, feel free to ignore it.

Reply:
This doesn't seem right to me.... Not a native speaker though. If you feel what I've got now is incorrect I can ask someone who's trained for this stuff.

Reply:
Although either is acceptable English, an online check seems to prefer 'correspond to' for things like objects, laws or map co-ordinates, while 'correspond with' is for things like communicating or moving with another person.